src/pages/signin/SignInPageLayout/BackgroundImage/index.js

```diff
@@ -1,32 +1,54 @@
 import PropTypes from 'prop-types';
 import React from 'react';
+import * as Animatable from 'react-native-animatable';
 import DesktopBackgroundImage from '@assets/images/home-background--desktop.svg';
 import MobileBackgroundImage from '@assets/images/home-background--mobile.svg';
 import useThemeStyles from '@hooks/useThemeStyles';
 import defaultPropTypes from './propTypes';
 
 const defaultProps = {
     isSmallScreen: false,
+    transitionDuration: 0,
 };
 
 const propTypes = {
     /** Is the window width narrow, like on a mobile device */
     isSmallScreen: PropTypes.bool,
 
+    /** Tranistion duration in milisecond */
+    transitionDuration: PropTypes.number,
+
     ...defaultPropTypes,
 };
 function BackgroundImage(props) {
     const styles = useThemeStyles();
-    return props.isSmallScreen ? (
-        <MobileBackgroundImage
-            width={props.width}
-            style={styles.signInBackground}
-        />
-    ) : (
-        <DesktopBackgroundImage
-            width={props.width}
+    const fadeIn = {
```

Comment:
Let's move this outside the `BackgroundImage` function

Reply:
@eVoloshchak we are not using `Animatable.View` for native, and `expo-image` [ImageTransition](https://docs.expo.dev/versions/latest/sdk/image/#imagetransition) has a different param structure.

Should we :

1. Continue extracting the animation prop but no need to use it for native
2. Or use the same `<Animatable.View>` for native?

Reply:
@arosiclair thoughts on it?

Reply:
Not sure why we would need to define `fadeIn` somewhere else. Seems fine here to me.

Reply:
@arosiclair, @eVoloshchak  do we agree to keep it this way?